config/vehicle_info.param.yaml

```diff
@@ -1,12 +1,12 @@
 /**:
   ros__parameters:
     wheel_radius: 0.39
-    wheel_width: 0.42
-    wheel_base: 2.74 # between front wheel center and rear wheel center
-    wheel_tread: 1.63 # between left wheel center and right wheel center
+    wheel_width: 0.235
+    wheel_base: 2.79 # between front wheel center and rear wheel center
+    wheel_tread: 1.64 # between left wheel center and right wheel center
     front_overhang: 1.0 # between front wheel center and vehicle front
-    rear_overhang: 1.03 # between rear wheel center and vehicle rear
-    left_overhang: 0.1 # between left wheel center and vehicle left
-    right_overhang: 0.1 # between right wheel center and vehicle right
+    rear_overhang: 1.1 # between rear wheel center and vehicle rear
+    left_overhang: 0.13 # between left wheel center and vehicle left
+    right_overhang: 0.13 # between right wheel center and vehicle right
```

Comment:
Is this `(1895 - 1640) / 2` = 127.5 mm = 0.13 m?

Reply:
yes.

Reply:
Seeing this, should this be `0.1275` or `0.128`? (I personally feel 3 digits are enough.)
![image](https://user-images.githubusercontent.com/31987104/167393398-11ec4149-573a-4bdf-9ad3-a7c26806eaae.png)

Reply:
@yukkysaito Any comments?

Reply:
Right. Aligned with others, I changed it to 0.128